code-samples/eventing/bookstore-sample-app/db/sample.sql

```diff
@@ -0,0 +1,12 @@
+CREATE TABLE BookReviews(

```

Comment:
We should verify that the table does not exist before creating it. Also small nit on the naming of the table

```suggestion
CREATE TABLE IF NOT EXISTS book_reviews(

```